quota/etcd/quotapb/gen.go

```diff
@@ -14,4 +14,4 @@
 
 package quotapb
 
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/googleapis/googleapis --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --go-grpc_opt=require_unimplemented_servers=false quotapb.proto
+//go:generate protoc -I=. -I=../../../third_party/googleapis --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --go-grpc_opt=require_unimplemented_servers=false quotapb.proto
```

Comment:
You might be able to do something like `... -I=$(go list -f '{{.Root}}' .)/third_party/googleapis ...` to avoid ever having to update the relative path.

Reply:
It's a nice idea, but failing:
```
protoc -I=. -I=$(go list -f '{{.Root}}' .)/third_party/googleapis --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --go-grpc_opt=require_unimplemented_servers=false quotapb.proto
$(go: warning: directory does not exist.
Unknown flag: -f
gen.go:17: running "protoc": exit status 1
```

Now if I hadn't spent a huge amount of time trying to work out the magic of these go:generate protoc invocations before I'd spend the "couple of minutes" looking at this. But I know it's a trap :-)

And yeah, I confirmed that running `go list -f '{{.Root}}' .` naked in a shell does the right thing.

Reply:
I guess you'd have to explicitly invoke a shell along the lines of `go:generate bash -c "protoc -I=$(go list ...) ...` to make it work. I don't know whether that's a net win or not...

Reply:
There is a precedent here: https://github.com/google/certificate-transparency-go/blob/1a8898f9bd49798a7a442d3691fdc79c6c31807f/proto_gen.go#L23

Reply:
hehe, turns out it was me :)

Reply:
Oh wow, this works by invoking a shell to invoke protoc. It's cool that this is possible, in a weird way. But are we really sure that the relative paths are _that bad_ ? If we were moving this around a lot, I can see the argument. If this wasn't covered by presubmits then that would add further weight to it.

I think I'm particularly resistant because I've spent a lot of time in the last week debugging shell scripts and it has been whatever the opposite of fun is.

Reply:
I am 80/20 in favor of the simpler command, at the cost of a relative path.

Reply:
I'll keep this simpler invocation with the relative paths.